app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] According to the [ERD Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), associations between models are set up correctly, something like so: 

```ruby
belongs_to :author, class_name: 'User'
has_many :comments
has_many :likes
```
Please kindly fix that.

Reply:
@t-yanick  Thanks fixed.